packages/rocket-uploads3-store-event-aws-infrastructure/src/s3-notification-event-store.ts

```diff
@@ -0,0 +1,68 @@
+import { Duration, RemovalPolicy, Stack } from '@aws-cdk/core'
+import { Bucket, EventType } from '@aws-cdk/aws-s3'
+import { RocketUtils } from '@boostercloud/framework-provider-aws-infrastructure'
+import { Function, Runtime, Code } from '@aws-cdk/aws-lambda'
+import { Table } from '@aws-cdk/aws-dynamodb'
+import { S3EventSource } from '@aws-cdk/aws-lambda-event-sources'
+import { createPolicyStatement } from '@boostercloud/framework-provider-aws-infrastructure/dist/infrastructure/stacks/policies'
+
+const path = require('path')
+
+export type AWSS3EventsLambdaParams = {
+  bucketName: string
+  eventTypeName: string
+  entityTypeName: string
+}
+
+export class S3NotificationEventStoreStack {
+  public static mountStack(params: AWSS3EventsLambdaParams, stack: Stack): void {
+    const appName = stack.stackName.split('-app')[0]
+
+    // Create new bucket to upload file
+    const sourceBucket = new Bucket(stack, 'sourceUploadBucket' + params.entityTypeName, {
+      bucketName: params.bucketName,
+      removalPolicy: RemovalPolicy.DESTROY,
+    })
```

Comment:
Ideally, an existing user should be able to upload files directly from their client application to S3. I didn't find a policy that allows this in here, so how does this work? Also, it would be ideal to integrate this module with Booster's role system, so we can grant upload permissions for a user role but not to another, or avoid anonymous users to upload a file. Any ideas around that? (We could perfectly leave this for a future PR)

Reply:
` an existing user should be able to upload files directly from their client application to S3` => Not sure I'm following. As of now, when we deploy our booster project, we can navigate to the AWS console and then upload the file to be processed there.

` it would be ideal to integrate this module with Booster's role system` => Yes! Totally makes sense. Let's tackle this on a separate PR :)

Reply:
Gotcha, we don't need to add this functionality in this PR, but I think that it would make a lot of sense to integrate this rocket with Booster's roles to also provide an API that someone could use to upload a file from a web browser. As it is today, it's restricted to users that can authenticate in the AWS console.